third_party/rust-on-exercism/health-statistics.rs

```diff
@@ -6,26 +6,32 @@ struct User {
 
 impl User {
     pub fn new(name: String, age: u32, weight: f32) -> Self {
+        println!("Use name {:?} age {:?} and weight{:?}", name, age, weight);
```

Comment:
I've tried to inline the format strings everywhere else in the course:

```suggestion
        println!("Use name {name:?} age {age:?} and weight {weight:?}");
```

I also fixed a space, which makes me think that you've not looked at this in the browser after making the change?